MPC/config/dcps_qos_xml_handler.mpb

```diff
@@ -1,5 +1,5 @@
-// -*- MPC -*-
-
-project : dcps_qos_xml_handler_base, dynamicany, typecodefactory {
-  requires += dds4ccm_opendds
+project : dynamicany, typecodefactory, dcps {
```

Comment:
Are TAO's dynamic any and typecode factory libraries actually required?  They aren't listed in qos_xml_handler.mpc.